### DISCUSSION TITLE
Como gerar um .exe que depende de vários "arquivo.py" ?

### DISCUSSION THREAD

**Discussion by eduardoSilva96** [marcelotduarte/cx_Freeze#1680] [General]
Fiz um sistema que depende de outros arquivo.py
Porém não estou conseguindo ter sucesso.
Sendo "treatment" e "manipulation" meus .py
Meu setup :

```
import sys
from cx_Freeze import setup, Executable

build_exe_options = {
    "packages": [
        "os"
    ],
    "includes": [
        "tkinter",
        "treatment",
        "manipulation"
    ]
}

base = "Win32GUI"
if sys.platform == "win32":
    base = "Win32GUI"

setup(
    name="SCM_Teste",
    version="0.1",
    description="teste",
    options={"build_exe": build_exe_options},
    executables=[Executable("main.py", base=base)],
)

```

**Comment by marcelotduarte** [comment]
Se main.py importar os outros de acordo, vc não precisará usar eles nos includes. Alias, vc tem que conseguir rodar seu app usando python main.py, se rodar, será compiado e rodará.
Agora se vc qiuer três executaveis independentes, tbm é possível.
Como é o cabeçalho do seu main.py (imports) e como está a estruturado os outros 2 arquivos (junto ao main, em diretório)?

**Reply by eduardoSilva96** [reply]
Eu tirei eles do include, porém o erro permanece, estou importando os arquivos python da maneira certa na main.py

<img width="309" alt="image" src="https://user-images.githubusercontent.com/100139625/201495079-8c439ea0-04a5-43fc-bdce-5cef3579b87d.png">

**Reply by marcelotduarte** [reply]
Se vc rodar `python main.py` ele roda? Pq está dando um erro em main linha 70... Se fosse um modulenotfound ai poderia ser q não encontrou algum modulo...

**Reply by eduardoSilva96** [reply]
sim sim, o main.py roda de boa, que é a parte onde tem minhas inputs, e informo as planilhas que o sistema deve procurar;
Após eu apertar o botão de "baixar resultado", é aí aonde ele ja vai no treatment.py para fazer o tratamento das planilhas informadas, e em seguida direciona para as funções do manipulation.py, pra fazer a comparação e fazer o download do resultado em uma nova planilha.
